api/src/main/resources/liquibase.xml

```diff
@@ -8,6 +8,12 @@
         See http://www.liquibase.org/manual/home#available_database_refactorings
         for a list of supported elements and attributes
     -->
+	<changeSet id="ugandaemr-07032017-1958" author="ssmusoke">
+		<comment>Delete coded obs with null values from the summary page - most of these seem to be health education related concepts which are not on the summary page</comment>
+		<sql>
+			DELETE FROM obs WHERE concept_id IN (99169, 99170, 99171,99172, 99173,99174, 99175,99176,99177,99178,99179,99180,99181) AND value_coded IS NULL
```

Comment:
Do you have any idea how these got saved in the first place, with just NULL?
Or was the value saved or accidentally saved in another field like value_numeric, value_boolean, etc?